cmake/onnxruntime_unittests.cmake

```diff
@@ -525,6 +533,10 @@ else()
 endif()
 set_target_properties(onnxruntime_perf_test PROPERTIES FOLDER "ONNXRuntimeTest")
 
+if (onnxruntime_ENABLE_LANGUAGE_INTEROP_OPS)
+  target_link_libraries(onnxruntime_perf_test PRIVATE onnxruntime_language_interop onnxruntime_pyop)
```

Comment:
If onnxruntime_perf_test  dynamic link to onnxruntime, it shouldn't link to onnxruntime_pyop again. Otherwise, it will have duplicated symbols.

Reply:
onnxruntime_perf_test statically linked to onnxruntime_session which depends on onnxruntime_language_..., with not this line, build fails.

Reply:
Further, would u mind point me to the place where "dynamic link" happens?

Reply:
>  where "dynamic link" happens?
When onnxruntime_BUILD_SHARED_LIB is ON

Reply:
Excluded on  onnxruntime_BUILD_SHARED_LIB, thanks!